packages/bitcore-node/test/integration/matic/p2p.spec.ts

```diff
@@ -85,7 +85,7 @@ describe('Polygon', function() {
     const addresses = await wallet.getAddresses();
     expect(addresses).to.exist;
     expect(addresses.length).to.eq(1);
-    expect(addresses[0].toLowerCase()).to.equal('0xb875c670b079cce8f6ab4f013bd471359c877ac0');
+    expect(addresses[0].toLowerCase()).to.equal('0xa4e131D8c33FC059E9d245489DB03a4A61A2F32B');
```

Comment:
Checkout packages/crypto-wallet-core/src/derivation/paths.ts for a comment that explains this otherwise questionable change.